recipes/unicore/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -x -e
+
+PWD="$(pwd)"
+SOURCE_CFG="${PWD}/path.cfg"
+TARGET_CFG="${PREFIX}/etc/path.cfg"
+SOURCE_PY="${PWD}/src/py/predict_3Di_encoderOnly.py"
+TARGET_PY="${PREFIX}/etc/predict_3Di_encoderOnly.py"
+
+mkdir -p "${PREFIX}/etc"
+cp ${SOURCE_CFG} ${TARGET_CFG}
+cp ${SOURCE_PY} ${TARGET_PY}
+
+case $(uname) in
+	Darwin) rustup default stable && rustup target add x86_64-apple-darwin ;;
```

Comment:
Why is this needed ?
'compiler("rust")' doesn't do it ?!

Reply:
I encountered this error from the [previous check](https://github.com/bioconda/bioconda-recipes/actions/runs/12424337180/job/34689755774):
```
02:50:17 BIOCONDA INFO (ERR) error[E0463]: can't find crate for `std`
02:50:17 BIOCONDA INFO (ERR)   |
02:50:17 BIOCONDA INFO (ERR)   = note: the `x86_64-apple-darwin` target may not be installed
02:50:17 BIOCONDA INFO (ERR)   = help: consider downloading the target with `rustup target add x86_64-apple-darwin`
02:50:17 BIOCONDA INFO (ERR) 
02:50:17 BIOCONDA INFO (ERR) error[E0463]: can't find crate for `core`
02:50:17 BIOCONDA INFO (ERR)   |
02:50:17 BIOCONDA INFO (ERR)   = note: the `x86_64-apple-darwin` target may not be installed
02:50:17 BIOCONDA INFO (ERR)   = help: consider downloading the target with `rustup target add x86_64-apple-darwin`
02:50:17 BIOCONDA INFO (ERR) 
02:50:17 BIOCONDA INFO (ERR) For more information about this error, try `rustc --explain E0463`.
02:50:17 BIOCONDA INFO (ERR) error: could not compile `proc-macro2` (build script) due to 1 previous error
```

Reply:
let me [check](https://github.com/bioconda/bioconda-recipes/pull/52879/commits/9cd96c81d17c521b2bcd63b179707246c0c08357) whether it was temporary

Reply:
Guess it was temporary, weird 🤷‍♂️
It should now ready to be merged I suppose, please check :)